src/backend/distributed/sql/citus--11.0-4--11.1-1.sql

```diff
@@ -5,14 +5,16 @@
 #include "udfs/citus_internal_delete_partition_metadata/11.1-1.sql"
 #include "udfs/citus_copy_shard_placement/11.1-1.sql"
 
+-- The udf worker_fetch_foreign_file and worker_repartition_cleanup can cause upgrade to 11.1-1 to fail because of the different upgrade paths 
+-- that might have been taken by customers until the point 8.0-1. As a fix, we only drop the udfs if they exist.
```

Comment:
Minor:

> that might have been taken by customers until the point 8.0-1

lets be specific about what version caused problem for which UDF. That's a good  documentation